rootston/layer_shell.c

```diff
@@ -396,7 +396,12 @@ void handle_layer_shell_surface(struct wl_listener *listener, void *data) {
 			wlr_output_layout_output_at(desktop->layout,
 					seat->cursor->cursor->x,
 					seat->cursor->cursor->y);
-		assert(output); // And this one
+		if (!output) {
+			wlr_log(L_ERROR, "Couldn't find output at (%.0f,%.0f)",
+				seat->cursor->cursor->x,
+				seat->cursor->cursor->y);
+			output = wlr_output_layout_output_at(desktop->layout, 0, 0);
```

Comment:
Better to use `get_center_output`?

Also there might be zero outputs, in which case it would be wise to close the layer surface immediately.